src/main/java/com/karankumar/bookproject/backend/entity/Author.java

```diff
@@ -15,11 +15,7 @@
 
 package com.karankumar.bookproject.backend.entity;
 
-import lombok.AccessLevel;
-import lombok.AllArgsConstructor;
-import lombok.Data;
-import lombok.EqualsAndHashCode;
-import lombok.NoArgsConstructor;
+import lombok.*;
```

Comment:
Could you please revert this change? We are not using wildcard imports (see https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports)

Reply:
Done :slightly_smiling_face: